app/controllers/repo_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RepoController < ApplicationController
+  include RepoHelper
+  before_action :authenticate
+
+  def index
+    @projects, @pagination = get_projects_of_user(@current_user, params[:page])
```

Comment:
Using classes over methods is preferred in this case.
https://blog.engineyard.com/keeping-your-rails-controllers-dry-with-services